docs/pipelines/sci.md

```diff
@@ -4,264 +4,219 @@ sidebar-position: 3
+- the temporal granularity of the observations are sufficient to accurately capture the behaviour of our application
 
-`embodied-carbon` does expect some specific values to be available in the `inputs` array. These include:
 
-```yaml
-device/emissions-embodied: # the embodied emissions for the entire component
-time-reserved: # time the component is used by the application
-device/expected-lifespan: # lifespan of the component in seconds
-resources-reserved: # proportion of the total component being allocated
-resources-total: # size of the component
-```
-Most of these values can be found in manufacturer documentation for specific processors and other hardware. In the present case, you can again provide some default values for a hypothetical system. You can assume the resource is a processor being used in a cloud virtual machine. In this case, the `resources-total` can be the total number of VCPUs for the processor and the `resources-allocated` can be the number of VCPUs actually being used by your application. Remembering back to the Teads curve example, you already have that information available to you in the form of the `vcpus-total` and `vcpus-allocated` fields, which you can pass by name as values to `resources-total and ` resources-reserved`.
+## Components
 
-Add the following to your `defaults` section:
+There is only one component in this example. It represents the entire application. The component pipeline looks as follows:
 
-```yaml
-device/emissions-embodied: 1533.120 # gCO2eq
-time-reserved: 3600 # 1hr in seconds
-device/expected-lifespan: 94608000 # 3 years in seconds
-resources-reserved: vcpus-allocated
-resources-total: vcpus-total
+```
+pipeline:
+  compute:
+    - interpolate
+    - cpu-factor-to-wattage
+    - wattage-times-duration
+    - wattage-to-energy-kwh
+    - calculate-vcpu-ratio
+    - correct-cpu-energy-for-vcpu-ratio
+    - sum-energy-components
+    - embodied-carbon
+    - operational-carbon
+    - sum-carbon
+    - sci
 ```
 
-## Step 4: Calculate operational carbon
+## Plugins
 
-So far, you have calculated the *embodied* carbon for your application, but your usage values are still expressed as units of energy. To calculate the `carbon` emissions that result from that energy consumption, you need to multiply your total energy by the carbon intensity of the electricity you consume. This value is known as the `operational-carbon`. In a real example, the grid carbon intensity could be a time-varying value that also depends on your physical location. However, here you will hardcode it for simplicity.
+### Interpolate
 
-```yaml
-grid/carbon-intensity
-```
+The `interpolate` plugin is used once. The instance is named `interpolate`. It is used to interpolate the curve relating CPU utilization and thermal-design-power factor so that the right value canb be retrived for the observed CPU utilization at each timestep.
```

Comment:
```suggestion
The `interpolate` plugin is used once. The instance is named `interpolate`. It is used to interpolate the curve relating CPU utilization and thermal-design-power factor so that the right value can be retrieved for the observed CPU utilization at each timestep.
```